sdk/src/trace/tracer_provider.cc

```diff
@@ -48,6 +48,11 @@ bool TracerProvider::Shutdown() noexcept
 {
   return GetProcessor()->Shutdown();
 }
+
+bool TracerProvider::ForceFlush(std::chrono::microseconds timeout) noexcept
+{
+  return GetProcessor()->Shutdown(timeout);
```

Comment:
Curious - do we want to call ForceFlush or Shutdown? (I thought the latter)

Reply:
As per the specs (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#forceflush):
`ForceFlush MUST invoke ForceFlush on all registered SpanProcessors.`

So ForceFlush() method needs to be  defined both for SpanProcessor and Exporter. This can be done as separate PR and in that case Shutdown() call can be removed for now from above.

Note: PR to update specs for adding ForceFlush in SpanExporter is still open ( https://github.com/open-telemetry/opentelemetry-specification/pull/1467 )

Reply:
Thanks for the catch, @reyang 

@lalitb I think we only support one `SpanProcessor` now. I'd prefer to add `ForceFlush` to `SpanExporter` in a separate PR.

Reply:
> I think we only support one `SpanProcessor` now.

That's fine, I think we could support multiple processors by using a de-mux processor.

Here are some examples:
* https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/CompositeProcessor.cs
* https://github.com/open-telemetry/opentelemetry-python/blob/99128b35d1a9cf2ecb04097c6a87d320a552a3e4/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L127